onnxruntime/core/providers/cpu/quantization/quantize_linear.h

```diff
@@ -1,45 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-#pragma once
```

Comment:
This header file was only included by `onnxruntime/contrib_ops/quantization/quantize_ops.cc`, which has been removed by this PR. Therefore, this header file is now unnecessary. Please refer to the following comment for details: https://github.com/microsoft/onnxruntime/pull/17015#discussion_r1324777230